tests/test_callbacks.py

```diff
@@ -57,6 +63,19 @@ def test_callbacks(tmp_path, model_class):
     if os.path.exists(log_folder):
```

Comment:
I think this one can be removed.

Reply:
This was a mishap, will remove with a forced push.